ee/codegen/src/__test__/serializers/vellum.test.ts

```diff
@@ -203,7 +203,7 @@ describe("vellum", () => {
         definition: null,
       };
 
-      const parsedData = PromptNodeSerializer.parse(data);
+      const parsedData = WorkflowNodeSerializer.parse(data);
```

Comment:
switching this to `WorkflowNodeSerializer` since the child no longer has the type field